lang/perl5/Portfile

```diff
@@ -209,6 +209,17 @@ foreach {perl5.v perl5.subversion perl5.revision perl5.rmd160 perl5.sha256 perl5
         post-build {
             reinplace -E {s/-arch [a-z0-9_]+//g} \
                 ${worksrcpath}/lib/Config_heavy.pl
+            reinplace -E {s/-mmacosx-version-min=[0-9.]+//g} \
+                ${worksrcpath}/lib/Config_heavy.pl
+            reinplace "s|${configure.cc}|/usr/bin/cc|g" \
```

Comment:
Will that work on < 10.9 where we switched to `clang-mp` as the default compiler? How will the rest of the flags cope with that different compiler? I mean: perl comes up with tons of flags depending on various checks it does when it's being built with the latest clang, but then those flags and feature checks will be applied to `gcc-4.2` when building the modules like `p5-tk`? (Maybe we come up with a proper patch for `cc` soon anyway.)

Maybe we could introduce `${prefix}/bin/cc-mp-default` at some point ;) ;) ;)

@kencu

Reply:
We still want to set `CC` explicitly when building perl modules in MacPorts. This just fixes certain build systems that like to check if the baked-in compiler works. (And it can't get any more broken than trying to use a nonexistent compiler.)